unit-tests/src/test/scala/java/nio/file/attribute/UserPrincipalLookupServiceSuite.scala

```diff
@@ -0,0 +1,27 @@
+package java.nio.file.attribute
+
+import java.nio.file.FileSystems
+
+object UserPrincipalLookupServiceSuite extends tests.Suite {
+  val lookupService = FileSystems.getDefault.getUserPrincipalLookupService
+
+  test("lookupPrincipalByName succeeds for `root` user") {
+    assert(
+      lookupService.lookupPrincipalByName("root").getName == "root")
+  }
+
+  test("lookupPrincipalByName throws exception for `gobbledygook` user") {
```

Comment:
To be friendly to developers running the test suite on systems other than Travis, the two
tests expected to throw should execute only on Travis.

Adapting the Travis detection  idiom kindly supplied by  @francisdb on SN gitter:
(needs scalafmt)
```
 val isTravis = System.getenv("TRAVIS") == "true"
 
  test("lookupPrincipalByName throws exception for `gobbledygook` user") {
      if (isTravis) {
          assertThrows[UserPrincipalNotFoundException](
          lookupService.lookupPrincipalByName("gobbledygook"))
     }
}
```

Reply:
Fixed.